README.md

```diff
@@ -2,34 +2,81 @@
 
 My personal website monorepo.
 
-
 #### Server Rendered Routes
+
 ```
 www.brongan.com/catscii
 www.brongan.com/analytics
 ```
 
 #### Client Rendered Routes
+
 ```
 www.brongan.com/ishihara
 www.brongan.com/game-of-life
 www.brongan.com/mandelbrot
 ```
 
 #### Dev Stack
-* R u s t 
-* yew
-* tokio
-* axum
-* planetscale 
-* fly.io
-* nix/crane
-* podman
-* sentry
-* honeycomb
+
+- R u s t
+- yew
+- tokio
+- axum
+- planetscale
+- fly.io
+- nix/crane
+- podman
+- sentry
+- honeycomb
 
 #### Project Layout
-* client/ contains the frontend.
-* server/ contains the backend.
-* shared/ contains files shared between them.
 
+- client/ contains the frontend.
+- server/ contains the backend.
+- shared/ contains files shared between them.
+
+#### Development Setup
+
+1. **Prerequisites**
+
+   - Rust toolchain
+   - [Just](https://github.com/casey/just) command runner
+   - [Trunk](https://trunkrs.dev/) and [wasm-pack](https://rustwasm.github.io/wasm-pack/) (will be automatically installed if missing)
```

Comment:
I don't like automatically installing things for people. They might use a different package manager.